docs/Coding-Guidelines/HybridCRT.md

```diff
@@ -131,3 +126,47 @@ all projects in the repository via this statement:
 ```
 
 This applies to all projects in the repository (product, test, ...).
+
+
+## C++/CX Special Considerations
```

Comment:
Lots of prose in this section spanning multiple steps. List those steps up front and focus more on "here's what you need to do" first. You can add commentary on WHY, but for those not interested in the deeper reasons and just want the cookbook to patch their project to work they should be able to read the initial directions and not sweat the explanation if they don't want to / need to / have the time / ....